joomla/joomla-cms
# Remove empty spaces in classes

Pull Request for Issue # .

### Summary of Changes
In the category blog, featured blog, Cassiopeia module chromes there are empty spaces if no class is applied. This PR removes those spaces

### Testing Instructions
1. Install Joomla with the demo blog content
2. Go to the homepage menu item "Home" and make sure to remove "Leading Article Classes" and "Article Classes"
3. Set "# Leading Articles" and "# Intro Articles" both to 1
4. Go to the homepage and see if the spaces are there
5. Apply the patch an see if the spaces are gone
6. Next apply classes an see if the required space is added
![blog-layout](https://user-images.githubusercontent.com/5610413/154308069-561e0f14-738e-40ab-88b3-4ebe2ace26ef.jpg)


Do the same as above with the menu item "Blog" in the "Main Menu Blog" menu

![Image8](https://user-images.githubusercontent.com/5610413/154308493-fe039afa-6342-41d3-8751-f6831b736720.jpg)

The frontend module "Latest Posts" also has an empty space in the class name
The module uses the "Module Style" noCard

![image](https://user-images.githubusercontent.com/5610413/154309366-9d94ae6e-ec8d-4ae1-a48e-86896b59f91f.png)

After applying the patch this space is gone.
Also try to add a module class suffix and see if the required space is added.

Next switch the "Module Style" to card
In the same place is also a space
![image](https://user-images.githubusercontent.com/5610413/154313322-f7b132ca-28df-47d2-a4dd-40e13b9d1898.png)

### Actual result BEFORE applying this Pull Request
There are spaces

### Expected result AFTER applying this Pull Request
Spaces only applied if there is a class present

### Documentation Changes Required
No
